core/src/test/java/org/kiwix/kiwixmobile/core/page/bookmark/viewmodel/BookmarkViewModelTest.kt

```diff
@@ -73,17 +68,12 @@ internal class BookmarkViewModelTest {
     every { zimReaderContainer.name } returns "zimName"
     every { sharedPreferenceUtil.showBookmarksAllBooks } returns true
     every { bookmarksDao.bookmarks() } returns itemsFromDb.distinctUntilChanged()
+    every { bookmarksDao.pages() } returns bookmarksDao.bookmarks()
     viewModel = BookmarkViewModel(bookmarksDao, zimReaderContainer, sharedPreferenceUtil)
   }
 
   @Test
-  fun `initial state is Initialising`() {
-    viewModel.state.test().assertValue(bookmarkState())
-  }
-
-  @Test
-  internal fun `ExitBookmarks finishes activity`() {
-    viewModel.effects.test().also { viewModel.actions.offer(Exit) }.assertValue(Finish)
+  fun `Initial state is initialising a bookmark state`() {
```

Comment:
I think more all we need to assert is that the method intialState returns our intialState